lib/generators/create-react-app.js

```diff
@@ -0,0 +1,45 @@
+const spawn = require("../spawn");
```

Comment:
Please use single quotes in JavaScript files. I think all of the uses in here should be single quotes.